assets/js/src/marketing_optin_block/attributes.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import { getSetting } from '@woocommerce/settings';
+
+const { defaultText } = getSetting('mailpoet_data');
```

Comment:
We try to refactor all `.js` files to typescript. All new files we create are in typescript. Could you have this file as `.ts` as well?